src/hapi/pipelines/app/__main__.py

```diff
@@ -64,6 +64,58 @@ def parse_args():
     return parser.parse_args()
 
 
+def combine_default(country, default):
+    country["input"] = country["input"] + default["input"]
+    country["output"] = country["output"] + default["output"]
+    country["output_hxl"] = country["output_hxl"] + default["output_hxl"]
+    return country
+
+
+def find_defaults(config):
+    default_list = []
+    for key in config:
+        key_components = key.split("_")
+        if len(key_components) == 2 and key_components[1] == "default":
+            default_list.append(key)
+    return default_list
+
+
+def find_matching_top_level_keys(config, default_key):
+    matching_top_level_keys = []
+    default_prefix = default_key.split("_")[0]
```

Comment:
Similar to the previous comment, you should just remove the part that says "_default" (maybe the easiest thing is a str.replace("_default", ""))